src/main/java/CommandSet/AddCommand.java

```diff
@@ -0,0 +1,104 @@
+package CommandSet;
+
+import Helper.Storage;
+import Helper.TaskList;
+import Helper.Ui;
+
+import Tasks.Task;
+import Tasks.Todo;
+import Tasks.Deadline;
+import Tasks.Event;
+
+import Exceptions.MissingTimeArgumentException;
+
+/**
+ * <h1>AddCommand<h1/>
+ * <p>
+ * AddCommand class adds tasks to the list of user tasks.
+ *</p>
+ *
+ * @author Saravanan Anuja Harish
+ */
+public class AddCommand extends Command {
+
+    // space
+    private static final String SPACE = " ";
+
+    // starting index of a list or a char in string.
+    private static final int START_INDEX = 0;
+
+    // used to differentiate between constructors
+    private static final int DUMMY_VARIABLE = 1;
+
+    // Stores the command TODO.
+    private static final String TODO = "TODO";
+
+    // Stores the command DEADLINE.
+    private static final String DEADLINE = "DEADLINE";
+
+    // Stores the command EVENT.
+    private static final String EVENT = "EVENT";
+
+    /**
+     * Constructor for AddCommand.
+     * returns an instance of addCommand.
+     */
+    AddCommand() {
+        super();
+    }
+
+    /**
+     * adds the task to taskList.
+     * @param taskList the list of user tasks.
```

Comment:
Should there be an empty line between the description and parameter section?

Reply:
Thanks for pointing out! Just now fixed it to include a line here, and in other relevant places.😄